src/main/scala/sparkz/core/network/SendingStrategy.scala

```diff
@@ -31,6 +33,17 @@ case object BroadcastTransaction extends SendingStrategy {
   }
 }
 
+case class BroadcastBlock(settings: NetworkSettings) extends SendingStrategy {
+  val maxBlockBroadcastPeers: Int = settings.maxForgerConnections +
+    (settings.maxOutgoingConnections + settings.maxIncomingConnections) / 2
```

Comment:
Yeah but then there's no prioritization anywhere, we would just broadcast to all connected peers. 
This is to decrease the load on the node when it is broadcasting-rebroadcasting blocks. The load will increase because we are adding a new connection pool for the forgers.
If you think this approach is bad - broadcasting to everyone is also fine by me

Reply:
I would remove it for two reasons:
- we are adding a separate connection pool for the forgers but  compared to now the  total amount of peers will be the same (a peer will either be a forger or not), so there should not be an increase of load
- if we randomly remove some peers from the broadcast list there is the theoretical case that a peer will not receive the new block from any other node he is connected to